A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html

```diff
@@ -7,58 +7,58 @@
+                        <text id="Value" class="Common Value" x="17%" y="85%">---</text>
+                        <circle id="Illuminator" r="7%" cx="73%" cy="62%" />
                     </svg>
                 </div>
 
                 <div id="Mode">
                     <svg width="100%" height="100%">
-                        <text id="HDG" class="Common Active" x="45%" y="50%" text-anchor="end" alignment-baseline="middle">HDG</text>
-                        <text id="TRK" class="Common Inactive" x="45%" y="85%" text-anchor="end">TRK</text>
-                        <rect id="separator" x="48%" y="0%" width="4%" height="100%" fill="rgb(3, 50, 126)" fill-opacity="0.15" />
-                        <text id="VS" class="Common Active" x="55%" y="50%" alignment-baseline="middle">V/S</text>
-                        <text id="FPA" class="Common Inactive" x="55%" y="85%">FPA</text>
+                        <text id="HDG" class="Common Active" x="40%" y="50%" text-anchor="end" alignment-baseline="middle">HDG</text>
+                        <text id="TRK" class="Common Inactive" x="40%" y="85%" text-anchor="end">TRK</text>
+                        <rect id="separator" x="47%" y="0%" width="6%" height="100%" fill="rgb(158, 158, 157)" fill-opacity="0.015" />
+                        <text id="VS" class="Common Active" x="60%" y="50%" alignment-baseline="middle">V/S</text>
+                        <text id="FPA" class="Common Inactive" x="60%" y="85%">FPA</text>
                     </svg>
                 </div>
 
                 <div id="AltVS">
-                    <div id="VerticalSpeed">
+                    <div id="Altitude">
                         <svg width="100%" height="100%">
-                            <text id="VS" class="Common Active" x="66%" y="25%" text-anchor="end">V/S</text>
-                            <text id="FPA" class="Common Inactive" x="93%" y="25%" text-anchor="end">FPA</text>
-                            <text id="Value" class="Common Value" x="21%" y="85%">-----</text>
-                            <circle id="DEC_PNT" r="1.5%" cx="56.5%" cy="82.5%" />
+                            <text id="ALT" class="Common Active" x="50%" y="25%">ALT</text>
+                            <text id="Value" class="Common Value" x="24%" y="85%">-----</text>
                         </svg>
                     </div>
 
-                    <div id="Altitude">
-                        <svg width="125%" height="100%">
-                            <text id="ALT" class="Common Active" x="40%" y="25%">ALT</text>
-                            <text id="Value" class="Common Value" x="20%" y="85%">-----</text>
-                            <circle id="Illuminator" r="5.6%" cx="85%" cy="62%" />
+                    <div id="VerticalSpeed">
+                        <svg width="100%" height="100%">
+                            <circle id="Illuminator" r="7%" cx="6%" cy="62%" />
```

Comment:
this circle is for the altitude, moving it here broke it so it always displays, I will fix.

Reply:
looks like a copy paste error that happened when you swapped the Altitude withe the Vertical speed svg, which I dont think was necessary

Reply:
OK. Fix it please